testcases/ock_tests.sh.in

```diff
@@ -288,12 +288,18 @@ main_script()
         LOGFILE=1
     fi
 
+    # Turn space separated list into array
+    totrace=($OCK_TRACE_TOKENS)
+
     for i in $SLOT; do (
         echo "********** Testing Slot $i **********"
         check_slot $i || { echo "SKIPPING slot $i"; exit; }
         if [ $NONEED_TOKEN_INIT -eq 0 ]; then
             init_slot $i || { echo "SKIPPING slot $i"; exit; }
         fi
+	for item in "${totrace[@]}"; do
```

Comment:
the indention seems wrong, or is this just the way it looks in github?

Reply:
You are right.  Tabs versus Spaces again.  Fixed.